tests/rust-integration-tests/runtimetest/src/tests.rs

```diff
@@ -76,3 +79,51 @@ pub fn validate_hostname(spec: &Spec) {
         }
     }
 }
+
+/// Run argument test recursively for files after base_dir
+fn do_test_mounts_recursive(base_dir: &Path, test_fn: &dyn Fn(&Path) -> Result<()>) -> Result<()> {
+    let rd = read_dir(base_dir).unwrap();
+    for d in rd {
+        let d = d.unwrap();
+        let f_type = d.file_type().unwrap();
+        if f_type.is_dir() {
+            do_test_mounts_recursive(d.path().as_path(), test_fn)?;
+        }
+
+        if f_type.is_file() {
+            test_fn(d.path().as_path())?;
+        }
+    }
+
+    Ok(())
+}
```

Comment:
1. don't use doc-comment for function, just use `//`
2. please improve naming,  eg, use `dirs` instead of `rd`, `dir` instead of `d` and so on

Reply:
fixed